.github/workflows/ci-core.yml

```diff
@@ -32,13 +32,18 @@ jobs:
         pip install --upgrade pip
     - name: Install the package locally
       run: pip install -e .[extras]
-    - name: Test with pytest and check code coverage
+    - name: Test with pytest and generate code coverage
       run: |
-        pytest --cov --cov-config=.coveragerc --cov-report=xml tests/
+        pytest --cov --cov-config=.coveragerc
     - name: Execute lit tests
       run: |
         export PYTHONPATH=$(pwd)
-        lit -v tests/filecheck/
```

Comment:
IMO, we might want to think about getting the test coverage from the MLIR-based CI instead of this one. It only takes around 4minuts more to run it and we would get better coverage data. Additionally, it usually takes the codecov bot around 5minutes to respond to a change, so essentially we would increase the time the bot needs to around 10 minutes, which I feel is not that much worse than 5 minutes. What do you think?

Reply:
I think it's fine to wait for 10 minutes for coverage! Most of the time, it takes longer to review anyway ;)

Reply:
Moved it :+1: